### PR TITLE
Update grammar and highlight queries for slint grammar rewrite

### PR DESCRIPTION
**Pull request by jrmoulton**
This is just updating the grammar rev and queries to be updated to the latest slint grammar.

### CODE REVIEW

**Comment by jrmoulton** [comment]
Ahh. I accidentally have an extra unrelated commit in here. I'll fix that

**Comment by jrmoulton** [comment]
Fixed

File: runtime/queries/slint/highlights.scm [hunk 1]
```diff
@@ -118,8 +148,7 @@
  "*="
  "/="
  "?"
-
  "=>"
  ] @operator
 
-(ternary_expression [":" "?"] @keyword.control.conditional)
\ No newline at end of file
+(ternary_expression [":" "?"] @conditional)
```
Comment (the-mikedavis) [inline runtime/queries/slint/highlights.scm]: Helix uses different captures for highlighting than neovim - https://docs.helix-editor.com/master/themes.html#syntax-highlighting- so this change should be undone

There are some other non-existent scopes that should be changed too, for example `@float` should become `@constant.numeric.float`

**Review by the-mikedavis — COMMENTED** [review]
It looks like `local.scm` will need to be updated as well since it contains a node that doesn't exist in the new version of the grammar: https://github.com/helix-editor/helix/actions/runs/3777270745/jobs/6422763958

**Comment by lauglam** [comment]
Has there been any progress on this issue?

**Comment by jrmoulton** [comment]
I haven’t worked on this in a while. There isn’t much that needs changed at this point but the queries need to be updated for helix

**Comment by gabydd** [comment]
Closed by https://github.com/helix-editor/helix/pull/7893

**Pull request closed without merging**